boa3/env.py

```diff
@@ -3,4 +3,12 @@
 PROJECT_ROOT_DIRECTORY = '/'.join(os.path.dirname(__file__).split(os.sep)[:-1])
 
 # If you didn't install TestEngine in this project's root folder, change this to the path of your .dll folder
-TEST_ENGINE_DIRECTORY = '{0}/Neo.TestEngine'.format(PROJECT_ROOT_DIRECTORY)
+TEST_ENGINE_DIRECTORY = ""
+
+TEST_ENGINE_DIRECTORY_DEFAULT = '{0}/Neo.TestEngine'.format(PROJECT_ROOT_DIRECTORY)
+TEST_ENGINE_DIRECTORY_ENV = os.getenv("BOA_TEST_ENGINE")
+
+if TEST_ENGINE_DIRECTORY_ENV is not None and len(TEST_ENGINE_DIRECTORY_ENV) > 0:
+    TEST_ENGINE_DIRECTORY = TEST_ENGINE_DIRECTORY_ENV
+else:
+    TEST_ENGINE_DIRECTORY = TEST_ENGINE_DIRECTORY_DEFAULT
```

Comment:
we use this to configure the CircleCI's unit tests workflow, it shouldn't be changed in feature pull requests
```suggestion
TEST_ENGINE_DIRECTORY = '{0}/Neo.TestEngine'.format(PROJECT_ROOT_DIRECTORY)
```

Reply:
It doesn't change default behavior, so should still work with CircleCi, no? I mean, I can always work around that, but it's pretty cumbersome. If the `TestEngine` needs to be in a specific place, then I would suggest to add a git submodule or something like that.